docs/source/about/metasynth_in_detail.rst

```diff
@@ -0,0 +1,270 @@
+Detailed overview of MetaSynth
+=================================
+
+``Metasynth`` is a python package for generating synthetic data with a
+focus on privacy and disclosure control. It is aimed at owners of
+sensitive datasets such as public organisations, research groups, and
+individual researchers who want to improve the accessibility of their
+data for research and reproducibility by others. The goal of
+``metasynth`` is to make it easy for data owners to share the structure
+and and approximation of the content of their data with others without
+any privacy concerns.
+
+With this goal in mind, ``metasynth`` restricts itself to
+the `\"synthetically-augmented plausible" <https://www.ons.gov.uk/methodology/methodologicalpublications/generalmethodology/onsworkingpaperseries/onsmethodologyworkingpaperseriesnumber16syntheticdatapilot>`__ category of synthetic data, as categorized by the Office for National Statistics (ONS).
+
+.. admonition:: ONS criteria for a Synthetically-augmented plausible dataset:
+   
+  * Preserve the format and record-level plausibility as detailed previously and replicate marginal (univariate) distributions where possible.
+  * Constructed based on the real dataset, values are generated based on observed distributions (with added fuzziness and smoothing) but no attempt made to preserve relationships.
+  * Missing value codes and their frequency is to be preserved.
+  * Disclosure control evaluation is necessary case by case, special care to be taken with names and so on.
+  * To be used for extended code testing, minimal analytical value, non-negligible disclosure risk.
+
+
+This choice enables the software to generate synthetic data with **privacy and disclosure guarantees** through a plug-in system. Moreover, our system provides an **auditable and editable intermediate representation** in the form of a human- and machine-readable ``.json`` metadata file from which new data can be synthesized.
```

Comment:
Yeah I was struggling to find a right placing for it, as it is now, it follows its placing in the JOSS paper. But I think it might be better to place it at the end of the page, so I'll do that for now. Later on (when this page is expanded upon) I will re-evaluate its placing.